screen-orientation/resources/sandboxed-iframe-locking.html

```diff
@@ -20,7 +20,12 @@
   }
 
   screen.orientation.unlock();
-  await document.exitFullscreen();
+  try {
+    await document.exitFullscreen();
+  } catch (error) {
+    data.result = "errored";
```

Comment:
Although FS generally only rejects with a `TypeError`, I wonder if it might be safer to pass `error.name`.

Reply:
Ok, I now pass error.name like in the try/catch above. Is this what you meant?